Source/Mosa.DeviceDriver/PCI/VMware/VMwareSVGA2.cs

```diff
@@ -19,6 +19,8 @@
 
 using System;
 using Mosa.DeviceSystem;
+using Mosa.DeviceSystem.Devices.Graphics;
+using Mosa.DeviceSystem.Drivers.Graphics;
```

Comment:
Duplicate.

Reply:
one is Devices.Graphics while the other is Drivers.Graphics